### DISCUSSION TITLE
Project goals

### DISCUSSION THREAD

**Discussion by floatdrop** [getbem/getbem.github.io#279] [Ideas]
Hi all!

I want to start discussion about goals of [getbem.com](https://getbem.com/) site and how we planning to achieve them.
# "Get BEM" to all people in simplest way

(no matter what size of project are they working on)

To achieve this we trying to separate BEM methodology from concrete libraries and tools, which will decrease threshold of entry in methodology. It is quite obvious, that BEM can be separated into Naming, Structuring and Building steps. Each of those can be used separately and give benefits of structured project or BEM naming in single index.css file.

Those articles are in pretty bad state for now—they have no illustrations, weak examples and very-very short. So we are looking for authors which share our point of view, that can help us to write those articles.
### What about bem.info?

We think, that [bem.info](https://bem.info/) have too many information and arcticles. Some of them are outdated, some of them are not translated to English. This leads to confusion and produces frustration, that associated with BEM methodology.
### What about bem-tools and ENB?

Bem-tools and ENB are concrete build-systems with they own advantages and disadvantages. Otherwise, we are focused on providing clear build instructions (for CSS, JavaScript and templates) which can be implemented in any build-systems.
### What about blocks’ libraries?

Concrete block libraries should be chosen and developed by project needs. There are not many ready to use blocks’ libraries out there, so we planning to provide instructions for developing and testing custom blocks libraries.

We think, that all this will help developers to create lots of great libraries for BEM community.

**Comment by varya** [comment]
I agree with all said :-) I think that as a result of this disscussion we need to set up a list of goals for getbem. Probably they are sub-goals of the main goal. And then we will be able to decide what can be our steps forward to those goals.

**Comment by varya** [comment]
I can start with listing the problems why BEM is not accepted yet in Europe/America according to my western experience. Needed?

**Comment by floatdrop** [comment]
@varya this is definitely must-know information! Could you create an issue with them?

**Comment by varya** [comment]
I can create a separate issue, but I cannot split all the problems into different issues. This would be to much work about structuring. As a first step I prefer to give the whole bunch of information together. Will do this soon, I created a task in my personal ToDo list.

**Comment by imalitavakoli** [comment]
@floatdrop great job man. I really loved the idea behind BEM and read almost all of the [bem.info](https://bem.info/) articles (of course the ones written in English!) and your website has been written in more simple terms and tried to explain everything easier to understand and faster to grasp for newbies!

Great job! I really believe that BEM deserves more to be known by more people and get scattered around the world!

**Comment by vinayak519** [comment]
I had read about BEM long back in some article, but lately i started applying in my CSS stylesheet and it really helps me to understand even more easily. Great work!

**Comment by acjay** [comment]
This site is great! 

Would people be opposed to adding a link to bem.info in the "Further Reading" section? It seems a bit strange to me that the website doesn't appear to acknowledge the original material on BEM. Even better would be some brief summary the purpose expressed above.

This came up in reality, because I'm teaching BEM to my team, and one of my colleagues was referencing this site, whereas I've been using BEM since the old days, and assumed they were using bem.info. There are some differences, such as the separator for introducing modifiers. He had the impression that this was the official source on BEM. We may well adopt this as our definitive guide, but it just made me a bit uncomfortable that the provenance of the idea of BEM isn't clearly stated here.

**Comment by rastapopougros** [comment]
+1 with @acjay it's not clear at all who maintained this site VS the official maintainers of BEM by yandex people.

All is always better with transparency, making it clear which is an enthusiasts site, and which is the official site (with the methodology evolving through time).

**Comment by floatdrop** [comment]
@rastapopougros @acjay If I would pick the one site for BEM – it most likely will be [bem.info](https://bem.info). Feel free to send PR to add it into Further reading section.
